package.json

```diff
@@ -4,66 +4,66 @@
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.57",
     "@material-ui/pickers": "^3.2.8",
-    "caniuse-lite": "^1.0.30001021",
-    "chart.js": "^2.9.3",
+    "caniuse-lite": "^1.0.30001192",
+    "chart.js": "^2.9.4",
     "classnames": ">=2.2.5",
-    "cropperjs": "^1.5.6",
-    "date-fns": "^2.6.0",
-    "downshift": "^4.0.6",
+    "cropperjs": "^1.5.11",
+    "date-fns": "^2.17.0",
+    "downshift": "^4.1.0",
     "history": "^4.10.1",
     "http-proxy-middleware": "^0.20.0",
-    "interweave": "^12.1.1",
-    "jwt-decode": ">=2.2.0",
-    "lodash": ">=4.17.11",
-    "luxon": "^1.21.3",
-    "moment": ">=2.24.0",
+    "interweave": "^12.7.0",
+    "jwt-decode": "^3.1.2",
+    "lodash": "^4.17.21",
+    "luxon": "^1.26.0",
+    "moment": "^2.29.1",
     "prop-types": ">=15.6.0",
-    "react": "^16.12.0",
-    "react-big-calendar": ">=0.21.0",
-    "react-chartjs-2": ">=2.7.4",
-    "react-cropper": ">=1.0.1",
+    "react": "^16.14.0",
+    "react-big-calendar": "^0.32.0",
+    "react-chartjs-2": "^2.11.1",
+    "react-cropper": "^2.1.7",
     "react-document-title": ">=2.0.3",
-    "react-dom": ">=16.2.0",
-    "react-dropzone": ">=10.1.5",
-    "react-ga": ">=2.3.5",
-    "react-graceful-image": ">=1.2.9",
-    "react-icons": ">=2.2.7",
-    "react-image-gallery": "^1.0.5",
+    "react-dom": "^17.0.1",
+    "react-dropzone": "^11.3.1",
+    "react-ga": "^3.3.0",
+    "react-graceful-image": "^1.5.0",
+    "react-icons": "^4.2.0",
+    "react-image-gallery": "^1.0.9",
     "react-media": ">=1.8.0",
-    "react-responsive-carousel": "^3.1.51",
+    "react-responsive-carousel": "^3.2.12",
     "react-router": "~5.0.1",
     "react-router-dom": "~5.0.1",
-    "react-timeago": ">=4.4.0",
-    "typescript": ">=3.5.3",
+    "react-timeago": "^5.2.0",
+    "typescript": "^4.2.2",
     "url-regex": ">=5.0.0"
   },
   "devDependencies": {
-    "eslint-config-react-app": "^5.1.0",
-    "eslint-plugin-flowtype": "^4.6.0",
-    "eslint-plugin-import": "^2.20.0",
-    "eslint-plugin-jsx-a11y": ">=5.1.1",
-    "eslint-plugin-react": "^7.18.0",
-    "husky": "^4.2.3",
-    "jasmine-core": "^3.5.0",
-    "jsdoc": ">=3.6.2",
+    "eslint-config-react-app": "^5.2.1",
+    "eslint-plugin-flowtype": "^4.7.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-react": "^7.22.0",
+    "husky": "^4.3.8",
+    "jasmine-core": "^3.6.0",
+    "jsdoc": "^3.6.6",
     "karma": "^4.4.1",
     "karma-chrome-launcher": "^3.1.0",
     "karma-firefox-launcher": "^1.3.0",
-    "karma-jasmine": "^3.1.0",
+    "karma-jasmine": "^3.3.1",
     "karma-safari-launcher": ">=1.0.0",
     "lint-staged": "^8.0.0",
-    "node-sass-chokidar": "^1.4.0",
+    "node-sass-chokidar": "^1.5.0",
     "npm-run-all": ">=4.1.2",
-    "prettier": ">=1.9.2",
-    "react-scripts": "^3.3.0",
-    "stylelint": ">=10.1.0",
+    "prettier": "^2.2.1",
+    "react-scripts": "^3.4.4",
+    "stylelint": "^13.11.0",
```

Comment:
Another two

Reply:
These are also devDependencies and cannot result in breaking changes.